client/src/components/graph/graph.tsx

```diff
@@ -793,6 +810,9 @@ class Graph extends React.Component<GraphProps, GraphState> {
       flagBuffer!({ data: flags, dimension: 1 });
       needToRenderCanvas = true;
     }
+    if (screenCap) {
+      needToRenderCanvas = true;
+    }
```

Comment:
I believe it is safe to not check if screencap is differeing from the last call of updateReglAndRender, since this is just an on off state. I put a log in this function and played around and it affirmed.